### DISCUSSION TITLE
LTE/4G hotspot

### DISCUSSION THREAD

**Discussion by giovi321** [RaspAP/raspap-webgui#1499] [Q&A]
Hi everyone, before I try to install raspap-webgui, could someone enlighten me on whether i can use my raspberry as a 4G/LTE hotspot?
I see there is an hotspot option in the software, but it also says that IP will be assigned by the upstream DHCP server that is in this case the 4G USB dongle that I will be using. However, the 4G dongle usually assigns only one ip address to the raspbery pi, and then the usb0 is bridget to wlan0 therefore making a bridged connection.
Would raspap-webgui be able to do this?

many thanks!

**Comment by billz** [comment]
4G dongles and many other devices are supported as sources of connectivity for your AP. 
Refer to https://docs.raspap.com/net-devices/

**Reply by giovi321** [reply]
Sorry, I didn't see the documentation page! It's very clear, thanks.

**Reply by billz** [reply]
No worries and you're very welcome!
